recipes/hsm/all/conanfile.py

```diff
@@ -0,0 +1,37 @@
+from conans import ConanFile, CMake, tools
+import os
+
+required_conan_version = ">=1.32.1"
+
+class HsmConan(ConanFile):
+    name = "hsm"
+    license = "MIT"
+    homepage = "https://github.com/erikzenker/hsm.git"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "The hana state machine (hsm) is a finite state machine library based on the boost hana meta programming library. It follows the principles of the boost msm and boost sml libraries, but tries to reduce own complex meta programming code to a minimum."
+    topics = ("state machine", "template meta programming")
+    requires = "boost/1.72.0"
+    no_copy_source = True
+    generators = "cmake"
```

Comment:
```suggestion
    generators = "cmake"
    settings = "os", "arch", "build_type", "compiler"
```
otherwise CMake generator cannot be deduced